OpenEdXMobile/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I think it's best to reset the keys and generate a new file, because it was up for a while for the whole Internet to see.